app/src/titleUtil.ts

```diff
@@ -0,0 +1,3 @@
+export const truncateTitle = (title: string, maxLength = 50): string => (title.length > maxLength
```

Comment:
i generally prefer not to have single line functions because refactoring them later introduces lots of changed lines with little relevance. would you mind changing it to have like => { return ... } please